server.py

```diff
@@ -550,6 +554,42 @@ def get(self):
 
         return return_dict
 
+class PendingMergesResource(restful.Resource):
+    def get(self):
+        pass
+
+    def put(self, region):
```

Comment:
(fyi PUT usually specifies an ID or identifier. if you're creating a brand new object with a newly assigned ID, it's a POST)